app/logic/emailHandler.py

```diff
@@ -71,11 +70,17 @@ def fetch_event_programs(self, program_id):
         """ Fetches all the programs of a particular event """
         # Non-student-led programs have "Unknown" as their id
         if program_id == 'Unknown' or program_id is None:
-            programEvents = ProgramEvent.select(ProgramEvent.program).where(ProgramEvent.event==self.event.id)
-            return [program.program for program in programEvents.objects()]
+            return self.event.program
         else:
             return [Program.get_by_id(program_id)]
```

Comment:
This function should return the same data type. Right now it's returning a list of program objects in one case and a single program object in another.